test/ufuzz.js

```diff
@@ -122,6 +129,8 @@ var TYPEOF_OUTCOMES = [
   'crap' ];
 
 var FUNC_TOSTRING = 'Function.prototype.toString=function(){return"function(){}"};';
```

Comment:
@kzc I shall wait on this PR before putting in https://github.com/mishoo/UglifyJS2/pull/1688#issuecomment-289293769 to avoid merge conflicts.